docs/Components/button.css

```diff
@@ -0,0 +1,220 @@
+    color: var(--white-only);
+}
+
+.bat-btn-light {
+    background-color: var(--lighter-primary);
+    color: var(--prussian-blue-only);
+}
+
+.bat-btn-dark {
+    background-color: var(--primary-red-dark);
+    color: var(--white-only);
+}
+
+
+.bat-btn-primary:hover {
+    background-color: var(--primary-red-dark);
+    color: var(--white-only);
+}
+
+.bat-btn-secondary:hover {
+    background-color: var(--prussian-blue);
+    color: var(--white);
+}
+
+.bat-btn-warning:hover {
+    background-color: var(--btn-warning-hover);
+    color: var(--prussian-blue);
+}
+
+.bat-btn-danger:hover {
+    background-color: var(--btn-danger-hover);
+    color: var(--white-only);
+}
+
+.bat-btn-success:hover {
+    background-color: var(--btn-success-hover);
+    color: var(--white-only);
+}
+
+.bat-btn-info:hover {
+    background-color: var(--btn-info-hover);
+    color: var(--white-only);
+}
+
+.bat-btn-light:hover {
+    background-color: var(--btn-light-hover);
+    color: var(--prussian-blue-only);
+}
+
+.bat-btn-dark:hover {
+    background-color: var(--primary-red-dark);
+    color: var(--white-only);
+}
+
+.bat-btn-primary-outlined {
+    color: var(--primary-red);
+    border: 2px solid var(--primary-red);
+}
+
+.bat-btn-secondary-outlined {
+    color: var(--prussian-blue);
+    border: 2px solid var(--prussian-blue);
+}
+
+.bat-btn-warning-outlined {
+    color: var(--btn-warning);
+    border: 2px solid var(--btn-warning);
+}
+
+.bat-btn-danger-outlined {
+    color: var(--btn-danger);
+    border: 2px solid var(--btn-danger);
+}
+
+.bat-btn-success-outlined {
+    color: var(--btn-success);
+    border: 2px solid var(--btn-success);
+}
+
+.bat-btn-info-outlined {
+    color: var(--btn-info);
+    border: 2px solid var(--btn-info);
+}
+
+.bat-btn-light-outlined {
+    color: var(--light-primary);
+    border: 2px solid var(--lighter-primary);
+}
+
+.bat-btn-dark-outlined {
+    color: var(--primary-red-dark);
+    border: 2px solid var(--primary-red-dark);
+}
+
+/* hover outlined */
+
+.bat-btn-primary-outlined:hover {
+    background-color: var(--primary-red);
+    color: var(--white-only);
+}
+
+.bat-btn-secondary-outlined:hover {
+    background-color: var(--prussian-blue);
+    color: var(--white);
+}
+
+.bat-btn-warning-outlined:hover {
+    background-color: var(--btn-warning);
+    color: var(--prussian-blue);
+}
+
+.bat-btn-danger-outlined:hover {
+    background-color: var(--btn-danger);
+    color: var(--white-only);
+}
+
+.bat-btn-success-outlined:hover {
+    background-color: var(--btn-success);
+    color: var(--white-only);
+}
+
+.bat-btn-info-outlined:hover {
+    background-color: var(--btn-info);
+    color: var(--white-only);
+}
+
+.bat-btn-light-outlined:hover {
+    background-color: var(--lighter-primary);
+    color: var(--prussian-blue-only);
+}
+
+.bat-btn-dark-outlined:hover {
+    background-color: var(--primary-red-dark);
+    color: var(--white-only);
+}
+
+.bat-btn-icon {
+    height: 3rem;
```

Comment:
Suggestion to use CSS variables with pre-defined `rem` values, can easily be used across all components!

Reply:
Okay, I'll try from next time.